tools/replay.go

```diff
@@ -0,0 +1,246 @@
+package main
+
+import (
+	"bufio"
```

Comment:
Gotta move this to a folder but the idea of runnable go files really entices me... There is `https://github.com/erning/gorun`, but it didn't work for my and I need to list my mod/sum files. I mean why? It's a script, I don't care about versioning. Otherwise it should be really just a few bash lines